calblink.go

```diff
@@ -911,6 +944,14 @@ func main() {
 	if *debugFlag {
 		debugOut = os.Stdout
 	}
+	clientSecretPath := "/path/to/client_secret.json"
+	credentials, err := loadClientCredentials(clientSecretPath)
```

Comment:
This should replace the io.ReadFile call on line 943 of the original file that reads in the client secret file.